src/GO/Job.php

```diff
@@ -498,7 +500,7 @@ private function finalise()
     /**
      * Email the output of the job, if any.
      *
-     * @return void
+     * @return bool
```

Comment:
Okay, I've only changed it because of the return true; I will remove it and change it back to void.